tests/bin/nodes.test.ts

```diff
@@ -150,7 +160,7 @@ describe('CLI Nodes', () => {
       expect(result.exitCode).toBe(0); // Succeeds.
       expect(result.stdout).toContain('Gestalt Invite');
       expect(result.stdout).toContain(remoteOnlineNodeId);
-    });
+    }, 100000);
```

Comment:
These timeouts should be removed. `100000` seconds? No test should run even close to this amount.

Reply:
Oh that was just to check something, I forgot to remove it.